### DISCUSSION TITLE
azure-web-apps-kudu-console-and-tcpping/

### DISCUSSION THREAD

**Discussion by giscus** [colindembovsky/colindembovsky.github.io#25] [Giscussions]
# azure-web-apps-kudu-console-and-tcpping/

I was working with a customer recently that put a website into Azure Web Apps. This site needed to connect to their backend databases (which they couldn’t move to Azure because legacy systems still needed to connect to it). We created an Azure VNet and configured site-to-site connectivity that created a secure connection between the Azure VNet and their on-premises network.

https://colinsalmcorner.com/azure-web-apps-kudu-console-and-tcpping/

**Comment by sbieri1** [comment]
Thank you for this article. I was looking for this. Now is there also a kudu command for telnet, nc or nmap to ckeck certain ports? As when I add the port, I get blank lines returned.
